migrations/20201009045236-viewing.ts

```diff
@@ -0,0 +1,36 @@
+import Base from "db-migrate-base";
+
+export async function up(
+  db: Base,
+  callback: Base.CallbackFunction
+): Promise<void> {
+  db.createTable(
+    "viewing_permissions",
+    {
+      viewer_id: {
+        type: "int",
+        notNull: false,
+        primaryKey: true,
+      },
+      viewee_id: "int",
+      relationship_type: "string",
+    },
+    callback
+  );
+}
+
+export async function down(
```

Comment:
similar to my comment for the up function, please add a comment above that describes what running down this migration would do